RevenueCatUI/Templates/V2/Components/Text/TextComponentViewModel.swift

```diff
@@ -261,25 +259,58 @@ struct TextComponentStyle {
 
 }
 
+enum GenericFont: String {
+
+    case serif, monospace, sansSerif = "sans-serif"
```

Comment:
@JayShortway @tonidero Does this casing for `sans-serif` match what Android has? I couldn't tell 😅

Reply:
Yes! This seems correct. FWIW, in Android, I believe we're using the font family `name`, which matches to `serif`, `monospace` and `sans-serif`.